pkgs/servers/teleport/default.nix

```diff
@@ -44,42 +43,26 @@ let
     OPENSSL_NO_VENDOR = "1";
 
     postInstall = ''
-      cp -r target $out
-    '';
-  };
-
-  roleTester = rustPlatform.buildRustPackage {
-    name = "teleport-roletester";
-    inherit version src;
-
-    cargoSha256 = "sha256-gCm4ETbXy6tGJQVSzUkoAWUmKD3poYgkw133LtziASI=";
-    buildAndTestSubdir = "lib/datalog/roletester";
-
-    PROTOC = "${protobuf}/bin/protoc";
-    PROTOC_INCLUDE = "${protobuf}/include";
-
-    postInstall = ''
-      cp -r target $out
+      cp ${buildAndTestSubdir}/librdprs.h $out/lib
     '';
   };
 
   webassets = fetchFromGitHub {
     owner = "gravitational";
     repo = "webassets";
-    rev = "67e608db77300d8a6cb17709be67f12c1d3271c3";
-    sha256 = "sha256-o4qjXGaNi5XDSUQrUuU+G77EdRnvJ1WUPWrryZU1CUE=";
+    rev = "eb34904732b6b23f7382dfbb20fed050a9369b31";
```

Comment:
v10.2.0 is supposed to include gravitational/webassets@789ca5098b3e1f66c0bea2d2014ae247b3c1f74f; see https://github.com/gravitational/teleport/tree/v10.2.0.

There’s now a v10.2.2 which uses gravitational/webassets@e42ac15dbc20b8e302187a4ad9cf2ce95df2e7ae.